modules/net-vpc-peering/variables.tf

```diff
@@ -27,6 +27,12 @@ variable "export_peer_custom_routes" {
   default     = false
 }
 
+variable "export_subnet_routes_with_public_ip" {
```

Comment:
can we rename to `export_public_ip_routes` for consistency?

Reply:
Sure, please check again

Reply:
I singned the CLA, which checks are you talking about?